libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -316,8 +316,7 @@ private AlarmReceiver obtainAlarmReceiver() {
     return new AlarmReceiver(new SchedulerCallback() {
       @Override
       public void onPeriodRaised() {
-        // TODO Remove after including UI sample app tests
-        System.out.println("MapboxTelemetry#onPeriodRaised");
+        // TODO Add more tests to AlarmReceiverTest
```

Comment:
Could we ticket this TODO instead of leaving it in code?

Reply:
Done 👉 https://github.com/mapbox/mapbox-events-android/issues/74